travis/install-openresty.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+set -eux
+
+PREFIX=${1:-$HOME/openresty}
+
+if [ ! -d "${PREFIX}/bin" ]; then
+  cd /tmp/
+  wget -T 60 -q -c http://openresty.org/download/${OPENRESTY_VERSION}.tar.gz
+  tar -xzf ${OPENRESTY_VERSION}.tar.gz
+  rm -rf ${OPENRESTY_VERSION}.tar.gz
+  cd ${OPENRESTY_VERSION}
+  ./configure --prefix="${PREFIX}" --with-luajit-xcflags=-DLUAJIT_ENABLE_LUA52COMPAT --with-debug
```

Comment:
what happens if a customer needs other modules to be compiled? can this be modified to add additional modules?